### PR TITLE
[Travis] Add latest JRuby version with JDK-11

### PR DESCRIPTION
**Pull request by prashantvithani**

### CODE REVIEW

**Comment by pitr-ch** [comment]
Thanks! I've updated it. We test only latest of each minor version. Also the older version are usually placed further down in the file.

**Comment by pitr-ch** [comment]
I've also added kept tests for both Java 8 and 11.

File: .travis.yml [hunk 1]
```diff
@@ -9,8 +9,11 @@ matrix:
   include:
     - name: MRI 2.5.1 Latest
       rvm: 2.5.1
-    - name: JRuby 9.2.0.0 Latest
-      rvm: jruby-9.2.0.0
+    - name: JRuby 9.2.4.0 Latest on Java 11
```
Comment (olleolleolle) [inline .travis.yml]:
```suggestion
    - name: JRuby 9.2.4.1 Latest on Java 11
```

File: .travis.yml [hunk 2]
```diff
@@ -9,8 +9,11 @@ matrix:
...
+      rvm: jruby-9.2.4.1
+      jdk: oraclejdk11
+    - name: JRuby 9.2.4.0 Latest on Java 8
```
Comment (olleolleolle) [inline .travis.yml]:
```suggestion
    - name: JRuby 9.2.4.1 Latest on Java 8
```

File: .travis.yml (hunk 1 above)
Comment (pitr-ch) [reply .travis.yml]: Thanks.

**Comment by pitr-ch** [comment]
Merging the commit into #780. The fix in #780 needs java 11 to be tested and this does not pass without it.

**Pull request closed without merging**